ru/turing/courses/lesson5/Main.java

```diff
@@ -1,7 +1,185 @@
 package ru.turing.courses.lesson5;
 
+import java.util.*;
+
 public class Main {
     public static void main(String[] args) {
         //todo вставить сюда сортировки пузырьком, сделать для каждой свой метод
+        int[] mass = new int[]{8, 4, 9, 15, 26, -1, 0};
+        bubbleSort(mass);
+
+        System.out.println("---------");
+        System.out.println();
+
+        String[] mass1 = new String[] {"abc", "efghj", null, "."};
+        bubbleSort(mass1);
+
+        System.out.println("---------");
+        System.out.println();
+
+
+        User u1 = new User("Nicky", 1111110000, 18);
```

Comment:
```suggestion
        User nicky = new User("Nicky", 1111110000, 18);
```
Здесь и дальше исправь нейминг на осмысленный